client-js/app/elements/labrad-grapher-dataset.html

```diff
@@ -18,15 +30,17 @@
   Polymer({
     is: 'labrad-grapher-dataset',
     properties: {
-      items: {
-        type: Array,
+      path: {
+        type: String,
         notify: true,
+      },
+      dataset: {
+        type: String,
+        notify: true
       }
     },
-    ready: function() {
-      this.items = [
-        'A single dataset'
-      ];
+    back: function(){
+      window.location.href='/grapher/'+this.path.toString().replace(/,/g,"/");
```

Comment:
Also, by setting the href here it causes a full page reload, which is unnecessary. Instead, we just want to transition to the previous view without reloading the app. For now, can use the same technique as the `app-link` element for this and emit an "app-link-click" event: `this.fire('app-link-click', {path: this.parentPath});` (replacing parentPath by whatever property name you decide on).